app/sagas/selectServer.js

```diff
@@ -112,6 +112,9 @@ const handleSelectServer = function* handleSelectServer({ server, version, fetch
 			serverInfo = yield getServerInfo({ server, raiseError: false });
 		}
 
+		// we'll set serverVersion as metadata for bugsnag
+		bugsnagServerVersion((serverInfo && serverInfo.version) || version);
```

Comment:
There's a duplicated use of `(serverInfo && serverInfo.version) || version` here.
Can we create a const?